gulpfile.js

```diff
@@ -65,10 +61,18 @@ function runCommand(command) {
 
 const WEBAPP_OUT = 'www';
 
-// Copies Babel polyfill from node_modules, as it needs to be included by cordova_index.html.
-function copyBabelPolyfill() {
+// Copies dependencies imported by [cordova/electron]_index.html.
+function copyIndexDependencies() {
   const babelPolyfill = 'node_modules/babel-polyfill/dist/polyfill.min.js';
-  return runCommand(`cp -v ${babelPolyfill} ${WEBAPP_OUT}/babel-polyfill.min.js`);
+  const webcomponentsEs5 = 'node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js';
```

Comment:
I'm not sure you're gaining much by having those variables.

Reply:
Good point